app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesViewHolder.kt

```diff
@@ -52,94 +53,63 @@ class PocketStoriesViewHolder(
-        stories?.let {
-            onStoriesShown(it)
+        Column(modifier = Modifier.padding(top = 72.dp)) {
+            SectionHeader(
+                text = stringResource(R.string.pocket_stories_header_1),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = horizontalPadding.dp)
+                    .wrapContentHeight(align = Alignment.Top)
+            )
+
+            Spacer(Modifier.height(17.dp))
+
+            PocketStories(
+                stories ?: emptyList(),
+                horizontalPadding.dp,
+                interactor::onStoryClicked,
+                interactor::onDiscoverMoreClicked
+            )
         }
     }
+}
 
-    Column(modifier = Modifier.padding(top = 72.dp)) {
-        SectionHeader(
-            text = stringResource(R.string.pocket_stories_header_1),
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(horizontal = horizontalPadding.dp)
-                .wrapContentHeight(align = Alignment.Top)
-        )
-
-        Spacer(Modifier.height(17.dp))
-
-        PocketStories(
-            stories ?: emptyList(),
-            horizontalPadding.dp,
-            onStoryClicked,
-            onDiscoverMoreClicked
-        )
-
-        Spacer(Modifier.height(24.dp))
-
-        SectionHeader(
-            text = stringResource(R.string.pocket_stories_categories_header),
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(horizontal = horizontalPadding.dp)
-                .wrapContentHeight(align = Alignment.Top)
-        )
-
-        Spacer(Modifier.height(17.dp))
-
-        PocketStoriesCategories(
-            categories = categories ?: emptyList(),
-            selections = categoriesSelections ?: emptyList(),
-            onCategoryClick = onCategoryClicked,
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(horizontal = horizontalPadding.dp)
-        )
-
-        Spacer(Modifier.height(24.dp))
-
-        PoweredByPocketHeader(
-            onLearnMoreClicked,
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(horizontal = horizontalPadding.dp)
-        )
+@Composable
+@Preview
+fun PocketStoriesViewHolderPreview() {
+    FirefoxTheme {
+        Column {
+            SectionHeader(
+                text = stringResource(string.pocket_stories_header_1),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 16.dp)
+                    .wrapContentHeight(align = Alignment.Top)
+            )
+
+            Spacer(Modifier.height(17.dp))
+
+            PocketStories(
+                stories = getFakePocketStories(POCKET_STORIES_TO_SHOW_COUNT),
+                contentPadding = 0.dp,
+                onStoryClicked = { _, _ -> },
+                onDiscoverMoreClicked = { }
```

Comment:
```suggestion
                onDiscoverMoreClicked = {}
```
This isn't quite a big deal. I was just searching through the AC/Fenix codebase to get a sense of what was more common. I mostly just want us to be cognizant about being consistent about some of these code styles, but ideally have a linter to enforce a style.